src/Swarm/TUI/View.hs

```diff
@@ -233,6 +238,14 @@ drawGameUI s =
   addCursorPos = case s ^. uiState . uiWorldCursor of
     Just coord -> topLabels . rightLabel ?~ padLeftRight 1 (drawWorldCursorInfo (s ^. gameState) coord)
     Nothing -> id
+  -- Add clock display in bottom left of the world view if focused robot
+  -- has a clock installed
+  addClock = case s ^. gameState . to focusedRobot of
```

Comment:
Could we also display `paused` when the game is not running, what ever the current focused robot is?

Reply:
Great idea, done! :heavy_check_mark:

Reply:
I also decided to show individual ticks in the clock display only when the game is paused, *i.e.* the decimal point and hex digit at the end of `0:00:03.a`.  That way you can see the tick count increase as you single-step the game.